src/styles/card.scss

```diff
@@ -0,0 +1,111 @@
+.card {
+  box-sizing: border-box;
+  width: 200px;
+  font-family: Roboto, sans-serif;
+  font-size: 10px;
+  font-weight: 400;
+  line-height: 14px;
+  border: 1px solid #f3f3f3;
+  border-radius: 5px;
+  padding: 32px 16px 16px;
+
+  transition: transform 300ms;
+
+  &__image {
+    display: block;
+    width: 160px;
+    height: 134px;
+    margin: 0 auto;
+  }
+
+  &__description {
+    width: 100%;
+    height: 36px;
+    margin: 40px 0 4px;
+    font-size: 12px;
+    font-weight: 500;
+    line-height: 18px;
+    color: $main-text-color;
+  }
+
+  &:hover {
+    cursor: pointer;
+    transform: scale(1.2);
+    .card__description {
+      color: #34568b;
+    }
+  }
+
+  &__product-code {
+    box-sizing: border-box;
+    width: 98px;
+    height: 14px;
+    margin: 0 0 16px;
+    color: $secondary-text-color;
```

Comment:
Ensure that the variable '$secondary-text-color' is defined in your variables file. This is important for maintaining consistency across your styles.